src/acceptance/api/basic_auth_test.go

```diff
@@ -0,0 +1,103 @@
+package api_test
+
+import (
+. "github.com/onsi/ginkgo"
+. "github.com/onsi/gomega"
+"net/http"
+"strings"
+)
+
+var _ = Describe("AutoScaler Basic Auth Tests", func() {
+
+	Context("API Server: basic auth tests", func() {
+		It("should succeed to check health", func() {
+			req, err := http.NewRequest("GET", healthURL, nil)
+			resp, err := DoAPIRequest(req)
+			Expect(err).ShouldNot(HaveOccurred())
+
+			defer resp.Body.Close()
+
+			Expect(err).ShouldNot(HaveOccurred())
+			Expect(resp.StatusCode).To(Equal(200))
+		})
+	})
+
+	Context("Eventgenerator: basic auth tests", func() {
```

Comment:
we should test the inverse of this, if the ops file is not enabled by default, then it doesn't make sense to assert that they are protected.  If we could change the test based on a configuration property to validate the output that would be best.